website/events/migrations/0064_event_slug.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 4.1.7 on 2023-02-22 20:07
+
+from django.db import migrations, models
+
+def slugText(str):
```

Comment:
camelCase is not python_style

Reply:
Also, https://docs.djangoproject.com/en/4.1/ref/utils/#django.utils.text.slugify actually guarantees a valid slug as output. This function would not output a valid slug if special characters/accents are in the input